### PR TITLE
Starter

### PR DESCRIPTION
**Pull request by PrettyPearl2302**
## Description

Setting up base for future website code
This step is highly necessary to have a strong base for future code. In this code snippet, the front end and database have been successfully connected to the backend.

## Milestones
currently working on building the home page.

## Resources
I used the code from Tim's sequelize demo app here. Link: https://github.com/timothy1ee/sequelize-demo 

## Test Plan
What it currently looks like:
first pull request

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import express from "express";
+const app = express();
+const port = process.env.PORT || 3000;
+
+app.get('/', (req, res) => {
```
Comment (ushanakg) [inline server.js:5]: assuming this isn't necessary and will get deleted as you fill the app out (we don't want to make a habit of keeping extraneous code)

**Comment by ushanakg** [comment]
I'm not sure if the title of your PR is the branch name or self selected, but either way a short sentence describing the PR is useful for reviewers since that's what shows up in the link preview!

**Review by ushanakg — APPROVED** [review]
Excited to see it take shape!

**Comment by jeffwdeng** [comment]
Good job keeping your commits well named and organized. I would suggest making smaller PRs next time -- it'll make it easier to review and understand context. But good job overall :)

File: server.js (hunk 1 above)
Comment (PrettyPearl2302) [reply server.js:5]: it is necessary actually!

**Pull request merged**